W&B v0.83.0 release notes (#2960)
wandb/docs: release-notes/server-releases.mdx

<EolReminder/>

<Update label="v0.83.0" description="July 22, 2026">
W&B Server 0.83.0 makes the Registry cross-platform accessible. On Dedicated Cloud, Weave users can now use the Registry without a Models seat, Weave prompts can be published to the Registry for versioning and reuse across an organization, and organization-level Registry access is managed through a new SCIM entitlement. This release also expands administrative controls over team visibility, service accounts, and usage.

## Support and end of life

- W&B Server v0.71 has reached end of life.

## Features

> - Team admins can now view team-level usage and billing charts, including an overages banner and searchable team selection, and set spend limits.

Author: ```suggestion
- Team admins can now view team-level [usage and billing](/platform/app/settings-page/billing-settings#plan-usage) charts, including an overages banner and searchable team selection, and set spend limits.
```

> - Team admins can now view team-level [usage and billing](/platform/app/settings-page/billing-settings#plan-usage) charts, including an overages banner and searchable team selection, and set spend limits.

Reviewer: 'spend alerts' NOT spend limits

- When searching for panels, you can now exclude panels whose names match a regex pattern from your search results. The filter remains active until you remove it, so you can use it to narrow a workspace down to the panels that matter most.
    <Frame>
    ![Excluding panels from search results with a regex pattern](/images/release-notes/v0-83-0/exclude-panels-by-regex.png)
    </Frame>

> - When you work with nested run groups, you can now color all the child groups under a parent group at once. Click the icon next to a parent run group's name to color every run in the hierarchy. You can distinguish parent groups at a glance while distinct colors flow down to all subgroups.

Author: ```suggestion
- When you work with nested [run groups](/models/runs/grouping#organize-runs-into-groups), you can now color all the child groups under a parent group at once. Click the icon next to a parent run group's name to color every run in the hierarchy. You can distinguish parent groups at a glance while distinct colors flow down to all subgroups.
```

    <Frame>
    ![Coloring all runs in a nested run group hierarchy with one click](/images/release-notes/v0-83-0/color-nested-run-groups.gif)
    </Frame>
    - If you turn off **Use grouping from the runs table in charts**, the color carries through to the individual lines, so you can see each run while keeping the parent group's color scheme intact.
    - Click **Reset group colors** to return to the default coloring.

> - In a workspace, you can now hide the sidebar directly from the page. The **Hide sidebar** control is now visible without opening the overflow (**...**) menu.

Author: ```suggestion
- In a workspace, you can now hide the sidebar directly from the page by clicking the **Hide sidebar** control (<Icon icon="fa-angle-double-right" iconType="solid" />) without first opening the action menu (<Icon icon="ellipsis" iconType="solid" />).
```

> - When viewing a large log file, click **Top** or press **Cmd+Home** to scroll to the oldest log, and click **Current** or press **Cmd+End** to scroll to the newest log.

Author: ```suggestion
- When viewing a large [log file](/models/app/console-logs), click **Top** or press **Cmd+Home** to scroll to the oldest log, and click **Current** or press **Cmd+End** to scroll to the newest log.
```

> - In a workspace, you can now hide the sidebar directly from the page by clicking the **Hide sidebar** control (<Icon icon="fa-angle-double-right" iconType="solid" />) without first opening the action menu (<Icon icon="ellipsis" iconType="solid" />).

Author: ```suggestion
- In a workspace, you can now hide the sidebar directly from the page by clicking the **Hide sidebar** control (<Icon icon="angle-double-right" iconType="solid" />) without first opening the action menu (<Icon icon="ellipsis" iconType="solid" />).
```

- Improved the responsiveness of organization membership tables, team project tables, and sweep tables.
- Improved the responsiveness and usability of the metric panel editor.
- Added baseline metrics for fork history recompute.
- You can now filter and search artifacts by virtual version aliases. Fuzzy matching isn't yet supported.
- Added counts of registries and artifacts per organization.

### Automations

These features are available in Dedicated Cloud only.

> - **Automation Hub** centralizes the creation and management of automations across projects and registries. Find it in the **Core** section of the sidebar.

Author: ```suggestion
- **Automation Hub** centralizes the creation and management of [automations](/models/automations#automations-overview) across projects and registries. Find it in the **Core** section of the sidebar.
```

- An automation can now be scoped to an organization, a team, a project, a registry, or a collection. This eliminates the need to duplicate an automation to achieve broad coverage. When creating an automation, click **Edit scopes**.

### Registry

These features are available in Dedicated Cloud only.

- All Models and Weave users in an organization now have Registry access by default. Existing per-registry permissions are unchanged. Administrators can revoke a user's organization-level Registry access through SCIM. See [Update Registry access](/platform/hosting/iam/scim#update-registry-access).

> - Weave prompts (`StringPrompt` and `MessagesPrompt`) can now be added to the Registry, so you can version, alias, and reuse prompts across your organization alongside model artifacts.

Author: ```suggestion
- Weave [prompts](/weave/guides/core-types/prompts#create-prompt-objects) (`StringPrompt` and `MessagesPrompt`) can now be added to the Registry, so you can version, alias, and reuse prompts across your organization alongside model artifacts.
```

> - Improved artifact collection overview editing and version grid selection. Auto-generate rich collection cards with dynamic artifact panels for more context.

Author: ```suggestion
- Improved [artifact collection](/models/registry/create_collection) overview editing and version grid selection. Auto-generate rich collection cards with dynamic artifact panels for more context.
```

### Weave

- Added a preconfigured signal menu to the **New agent** signal drawer.
- Token costs are now shown in the UI.
- Agent reactions (thumbs-up/thumbs-down) now save as tags instead of emoji reactions, so they appear alongside scorer tags and you can filter and chart them by agent.
- Signals and Playground agents are now hidden by default.

> - Moved dataset manipulation to the **Evaluations** page and improved the experience of creating datapoints.

Author: ```suggestion
- Moved dataset manipulation to the [**Evaluations** page](/weave/guides/core-types/evaluations) and improved the experience of creating datapoints.
```

> - Added an audio player to agent conversations, including a new **Play all** control that plays each file in sequence.

Author: ```suggestion
- Added an [audio](/weave/cookbooks/audio_with_weave#audio-with-weave) player to agent conversations, including a new **Play all** control that plays each file in sequence.
```

> - Improved the **Agents** tab.

Author: ```suggestion
- Improved the [**Agents**](/weave/custom-agents-quickstart) tab.
```

- Added a **Spans** minimap to agent conversations, with turn-anchored feedback data.
- Added copy controls to JSON embedded in user messages.
- Restored tags as the default agent signal type.
- Scoring-worker spans are now annotated with entity, project, scorer, and monitor details.
- In the **Agents** view, you can now link to individual spans within conversations.

## Performance improvements

- Improved the performance and efficiency of panel hovering and highlighting interactions.
- Improved the efficiency of workspace history key aggregation.
- Improved the performance of loading many artifacts in the **Artifacts** sidebar.
- Improved the throughput of the Weave scoring consumer loop.
- Improved performance and thread-pool sizing when Weave batch-inserts many worker judge-call spans.

## Security

**SCIM and access management**

> - Registry access is now a distinct entitlement, decoupled from Models seats. Administrators can grant or revoke a user's organization-level Registry access without changing their Models seat or Weave role. Setting `modelsSeat` to `none` no longer revokes Registry access when the user lacks the Weave role. See [Update Registry access](/platform/hosting/iam/scim#update-registry-access).

Reviewer: What is the point of this part of the sentance: "When the user lacks the Weave role."

Is: setting `modelsSeat` to `none` no longer revokes Registry access sufficient? 

or are we trying to say:

Setting `modelsSeat` to `none` no longer revokes Registry access, unless the Weave role is also set to none?
